frontend/src/pages/Home.js

```diff
@@ -1,7 +1,53 @@
+import { Link } from "react-router-dom";
+import VideoCard from "../components/VideoCard";
 import Wrapper from "../styles/Home";
 
 const Home = () => {
-  return <Wrapper>Recommended Videos</Wrapper>;
+  return (
+    <Wrapper>
+      <h1>Home</h1>
+      <h2>Categories</h2>
+      <div className="category">
+        <Link to="/feed/videos?genre=35" className="category-card">
+          <span>Comedy</span>
+        </Link>
+        <Link to="/feed/videos?genre=28">
+          <div className="category-card">
+            <span>Action</span>
+          </div>
+        </Link>
+        <Link to="/feed/videos?genre=27">
+          <div className="category-card">
+            <span>Horror</span>
+          </div>
+        </Link>
+        <Link to="/feed/videos?genre=10749">
+          <div className="category-card">
+            <span>Romance</span>
+          </div>
+        </Link>
+        <Link to="/feed/videos?genre=878">
+          <div className="category-card">
+            <span>Sci Fi</span>
```

Comment:
suggestion: you can crate a data array & can map it here for better readability.

Reply:
Thanks for pointing out, will do that